Modules/BenchPress.Azure/Public/CustomAssertions/BeInLocation.ps1

```diff
@@ -0,0 +1,60 @@
+function ShouldBeInLocation ($ActualValue, [string]$ExpectedValue, [switch] $Negate, [string] $Because) {
+  <#
+    .SYNOPSIS
+      Custom Assertion function to check status on a resource deployment.
+
+    .DESCRIPTION
+      BeInLocation is a custom assertion that checks the provided ConfirmResult Object for deployment success.
+      It can be used when writing Pester tests.
+
+    .EXAMPLE
+      $result = Confirm-AzBPResourceGroup -ResourceGroupName $rgName
+
+      $result | Should -BeInLocation westus3
+
+    .EXAMPLE
+      $result = Confirm-AzBPResourceGroup -ResourceGroupName $rgName
+
+      $result | Should -Not -BeInLocation westus2
+
+    .INPUTS
+      ConfirmResult
+      System.Switch
+      System.String
+
+    .OUTPUTS
+      PSCustomObject
+  #>
+  if ($null -eq $ActualValue){
+    [bool] $succeeded = $false
+
+    if ($Negate) { $succeeded = -not $succeeded }
+    $failureMessage = "Confirm result is null or empty."
+  }
+  else {
+    $resourceLocation = $ActualValue.ResourceDetails.Location
```

Comment:
Can you confirm that `Location` is at the root of every resource response object? For example, ensure that a VM's response (or any other resource response) isn't something like `$ActualValue.ResourceDetails.Metadata.Location`, where `$ActualValue.ResourceDetails.Location` would not exist. It seems like it should be at the root of every resource object, I just want to be sure that we do our due diligence and confirm it.

Reply:
Will deal with this in next issue #207